ddtrace/tracer/zipkinpayload.go

```diff
@@ -159,7 +155,7 @@ func convertLogs(logs []*logFields) []*sfxtrace.Annotation {
 
 func deriveKind(s *span) *string {
 	if kind, ok := s.Meta[spanKind]; ok {
-		return pointer.String(kind)
+		return pointer.String(strings.ToUpper(kind))
```

Comment:
Should it remove the entry from `s.Meta`?

Reply:
It's not added to the actual tags object: https://github.com/signalfx/signalfx-go-tracing/pull/41/files#diff-5432f3fab72a5c35e42f11ee804a347bR104

Reply:
Nice.